docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3'
+
+networks:
+    bibbox-default-network:
+      external: true
+
+services:
+
+  local-openspecimen-db:
+    image: mysql:8.0.26
+    container_name: local-openspecimen-db
+    restart: unless-stopped
+    networks:
+      - bibbox-default-network
+    environment:
+      - MYSQL_ROOT_PASSWORD=openspecimen
+      - MYSQL_DATABASE=openspecimen
+      - MYSQL_USER=openspecimen
+      - MYSQL_PASSWORD=openspecimen
+    volumes:
+      - ./data/mysql:/var/lib/mysql
+      - ./configs/openspecimen.cnf:/etc/mysql/conf.d/openspecimen.cnf:ro #read only 
+    cap_add:
+      - SYS_NICE 
+      
+  local-openspecimen-web:
+    build: 
+      context: .
+      dockerfile: Dockerfile
+    container_name: local-openspecimen-web
+    #user: tomcat
+    #privileged: true
```

Comment:
same here, otherwise you can delete line

Reply:
yes thx, removed it